services/web/server/src/simcore_service_webserver/reverse_proxy/handlers/paraview.py

```diff
@@ -19,7 +19,8 @@
 
 from ..settings import APP_SOCKETS_KEY
 
-SUPPORTED_IMAGE_NAME = "simcore/services/dynamic/3d-viewer"
+SUPPORTED_IMAGE_NAME = ["simcore/services/dynamic/3d-viewer",
```

Comment:
As an intermediate step, would it be possible to start adding a label in these images to identify the reverse proxy handler?

Reply:
yes, we should start writing a schema for these things.